Database setup fails on Ubuntu

### Version Information

### Environment

Linux (Ubuntu 20.04)

### What is the expected behaviour?
Database should get set up with the command

`npm run --prefix server db:reset`

Per Step 5 of the Contributing Guide here: [https://docs.tooljet.com/docs/contributing-guide/setup/ubuntu](https://docs.tooljet.com/docs/contributing-guide/setup/ubuntu)

### What is the current behaviour?
Command fails with the following error message:

```
Error during migration run:

src/services/data_sources.service.ts:1:24 - error TS2307: Cannot find module '@tooljet/plugins/dist/server' or its corresponding type declarations.

1 import allPlugins from '@tooljet/plugins/dist/server';
```

### How to reproduce the issue?

Follow setup steps on Contributing Guide for Ubuntu

### Please provide any traces or logs that could help here.

This is the output on Terminal right after running "npm run --prefix server db:reset" :

```

> server@0.0.1 db:reset
> npm run db:drop && npm run db:create && npm run db:migrate


> server@0.0.1 db:drop
> ts-node ./scripts/drop-database.ts

Dropped database tooljet_development

> server@0.0.1 db:create
> ts-node ./scripts/create-database.ts

Created database tooljet_development

> server@0.0.1 db:migrate
> npm run typeorm migration:run


> server@0.0.1 typeorm
> ts-node -r tsconfig-paths/register ./node_modules/typeorm/cli.js --config ormconfig.ts "migration:run"

query: SELECT * FROM current_schema()
Error during migration run:
src/services/data_sources.service.ts:1:24 - error TS2307: Cannot find module '@tooljet/plugins/dist/server' or its corresponding type declarations.

1 import allPlugins from '@tooljet/plugins/dist/server';
```

### Any possible solutions?

Problem seems to have started after plugins was moved around.

### If the bug is confirmed, would you be willing to submit a PR?

I am not familiar enough with the app right now to know how to resolve this.

--- a/docs/docs/contributing-guide/setup/ubuntu.md
+++ b/docs/docs/contributing-guide/setup/ubuntu.md
@@ -47,6 +47,7 @@ Follow these steps to setup and run ToolJet on Ubuntu. Open terminal and run the
     ```bash
     npm install --prefix server
     npm install --prefix frontend
+    npm run build:plugins
     ```
 
 5. Set up database

